Sources/LiveViewNative/Modifiers/Text Input and Output Modifiers/FontDesignModifier.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  FontDesignModifier.swift
+// LiveViewNative
+//
+//  Created by May Matyi on 4/24/23.
+//
+
+import SwiftUI
+
+/// Applies a font design to any element.
+///
+/// <Text modifiers={font_design(@native, design: :default)}>This text is default</Text>
+/// <Text modifiers={font_design(@native, design: :monospaced)}>This text is monospaced</Text>
+/// <Text modifiers={font_design(@native, design: :rounded)}>This text is rounded</Text>
+/// <Text modifiers={font_design(@native, design: :serif)}>This text is serif</Text>
+///
+/// ## Arguments
+/// * ``design``
+#if swift(>=5.8)
+@_documentation(visibility: public)
+#endif
+struct FontDesignModifier: ViewModifier, Decodable {
```

Comment:
If you put the availability check on the modifier type instead of in the body it should show more prominently in the docs.

```suggestion
@available(iOS 16.1, tvOS 16.1, watchOS 9.1, *)
struct FontDesignModifier: ViewModifier, Decodable {
```